modules/site-health/audit-autoloaded-options/load.php

```diff
@@ -116,3 +116,42 @@ function perflab_aao_autoloaded_options_size() {
 	global $wpdb;
 	return (int) $wpdb->get_var( 'SELECT SUM(LENGTH(option_value)) FROM ' . $wpdb->prefix . 'options WHERE autoload = \'yes\'' );
 }
+
+/**
+ * Fetches autoload top list.
+ *
+ * @since n.e.x.t
+ *
+ * @return array Autoloaded data as option names and their sizes.
+ */
+function perflab_aao_query_autoloaded_options() {
+	global $wpdb;
+
+	$option_threshold = apply_filters( 'perflab_aao_autoloaded_options_table_threshold', 100 );
```

Comment:
Add missing document for the filter.

```suggestion
	/**
	 * Filters max option value threshold to show option in Site Health.
	 *
	 * @since n.e.x.t
	 *
	 * @param int $option_threshold Autoloaded option value threshold. Default 100.
	 */
	$option_threshold = apply_filters( 'perflab_aao_autoloaded_options_table_threshold', 100 );
```